recipes/hugin/CMakeLists.txt

```diff
@@ -0,0 +1,956 @@
+
```

Comment:
Did you write this yourself?

Reply:
patching got really tiring with line endings. I made two small mods. I'll update the "patch"

Reply:
Ahh OK, I ask because I want to makes sure that it has the proper license/copyright statement at the top then

Reply:
Hopefully what I have is ok. I should probably use the full GPL language. Will do